cmd/query/app/apiv3/http_gateway_test.go

```diff
@@ -167,6 +167,11 @@ func TestHTTPGatewayGetTraceMalformedInputErrors(t *testing.T) {
 			requestUrl:    "/api/v3/traces/123?end_time=xyz",
 			expectedError: "malformed parameter end_time",
 		},
+		{
+			name:          "TestGetTraceWithInvalidRawTraces",
+			requestUrl:    "/api/v3/traces/123?query.raw_traces=foobar",
+			expectedError: "malformed parameter query.raw_traces",
```

Comment:
curious - why above error messages did not include `query.` prefix for param name?

Reply:
Those parameters don't have the `query` prefix - see https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/apiv3/http_gateway.go#L30-L32

Reply:
It's confusing to me why that's the case (this can be seen in the swagger file). But, regardless, according to swagger.json, when the url is `traces/?...` it's interpreted as `FindTraces` and all params have the `query.` prefix, but if the url is `traces/{traceID}?...` then it's the `GetTrace` method and the params don't have the prefix. Meaning that the above example is an invalid query.

Reply:
Got it! Thanks for the explanation - I fixed this now to have two constants.